cmd/codegen/generator/go/templates/module_funcs.go

```diff
@@ -378,6 +391,9 @@ type paramSpec struct {
 	// Only applies to arguments of type File or Directory.
 	// If the argument is not set, load it from the given path in the context directory
 	defaultPath string
+	// Only applies to arguments of type GitRepository or GitRef.
+	// If the argument is not set, load it from the given path in the context git
+	defaultGit string
```

Comment:
Note for me: it needs to add an option to the Elixir SDK to support this feature.